lib/ae_mdw/sync/server.ex

```diff
@@ -108,8 +120,23 @@ defmodule AeMdw.Sync.Server do
   defp process_state(s), do: s
 
   defp spawn_sync(from_height, to_height) do
+    from_mbi =
+      case Block.last_synced_mbi(from_height) do
+        {:ok, mbi} -> mbi + 1
+        :none -> -1
+      end
+
+    block_index = {from_height, from_mbi}
+    db_state = State.new()
+    from_txi = Block.next_txi()
+
     spawn_link(fn ->
-      Block.sync(from_height, to_height)
+      {blocks_mutations, _next_txi} = Block.blocks_mutations(block_index, from_txi, to_height)
```

Comment:
A nitpick as there is `from_txi` and `to_height` maybe instead of `block_index` could we separate it as `from_height` and `from_mbi`?

Reply:
Done